data/data/powervs/main.tf

```diff
@@ -6,34 +6,39 @@ provider "ibm" {
 
 module "bootstrap" {
   source            = "./bootstrap"
-  cloud_instance_id = var.cloud_instance_id
-  cluster_id        = var.cluster_id
+  cloud_instance_id = var.powervs_cloud_instance_id
+  cluster_id        = var.powervs_cluster_id
+  resource_group    = var.powervs_resource_group
+
+  cos_instance_location = var.powervs_cos_instance_location
+  cos_bucket_location   = var.powervs_cos_bucket_location
+  cos_storage_class     = var.powervs_cos_storage_class
+
+  bootstrap = var.powervs_bootstrap
+  sys_type  = var.powervs_sys_type
+  proc_type = var.powervs_proc_type
 
-  bootstrap = var.bootstrap
-  sys_type  = var.sys_type
-  proc_type = var.proc_type
   # TODO(mjturek): image and network IDs are not derived during terraform
   #                for other providers. Need to investigate and follow how
   #                other providers do this. cnorman's branch has some work
   #                towards this already.
-
-  image_name   = var.image_name
-  network_name = var.network_name
-}
-
-data "ibm_is_subnet" "vpc_subnet" {
-  name = var.powervs_vpc_subnet_name
+  image_name   = var.powervs_image_name
+  network_name = var.powervs_network_name
 }
 
-module "loadbalancer" {
-  source = "./loadbalancer"
-
-  cluster_id    = var.cluster_id
-  vpc_name      = var.powervs_vpc_name
-  vpc_subnet_id = data.ibm_is_subnet.vpc_subnet.id
-  bootstrap_ip  = module.bootstrap.bootstrap_ip
-
-  # TODO add resources for master/controller
-  master_ips = {}
-
-}
+# data "ibm_is_subnet" "vpc_subnet" {
+#   name = var.powervs_vpc_subnet_name
+# }
+# 
+# module "loadbalancer" {
+#   source = "./loadbalancer"
+# 
+#   cluster_id    = var.powervs_cluster_id
+#   vpc_name      = var.powervs_vpc_name
+#   vpc_subnet_id = data.ibm_is_subnet.vpc_subnet.id
+#   bootstrap_ip  = module.bootstrap.bootstrap_ip
+# 
+#   # TODO add resources for master/controller
+#   master_ips = []
+# 
```

Comment:
is this commented during the testing?

Reply:
Yep, commented this out to simplify test. Fixed.